src/drivers/gitlab.js

```diff
@@ -334,6 +334,8 @@ class Gitlab {
     const projectPath = await this.projectPath();
     const { issueId, report, id: commentId } = opts;
 
+    if (report.length >= 1000000) throw new Error('GitLab Comment too Large');
```

Comment:
How about creating a constant for this value (to avoid having a magic number)? Something like `MAX_COMMENT_LENGTH`?

We can put it on line 17, just before the class declaration:

```javascript
const API_VER = 'v4';
const MAX_COMMENT_LENGTH = 1000000;

class Gitlab {
```